app/pages/patient/patientinfo.js

```diff
@@ -374,12 +391,12 @@ var PatientInfo = React.createClass({
     }
 
     var birthday = formValues.birthday;
-    if (!(birthday && datetimeUtils.isValidDate(birthday))) {
+    if (!(birthday && sundial.isValidDate(birthday))) {
```

Comment:
will double check but it was from memory a straight swap from the isValidDate login in blip to the sundial version